wpeverest/everest-forms
# Enhance - Add bulk choice

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds a feature to add bulk options to `Checkbox`, `Multiple Choice` and `Dropdown` fields. This feature is activated only when you activate the EVF pro with license.

![image](https://user-images.githubusercontent.com/58679177/81760304-9451d000-94e6-11ea-9da0-e71cb504d42d.png)


### How to test the changes in this Pull Request:

1. Create a form and activate the EVF pro plugin
2. Add the three fields mentioned above
3. Now go to field options of those fields
4. Click on `Bulk Add` button, a textbox and a button with label `Presets` will appear
5. To add preset options click on the `Presets` button and choose one of the presets
6. To add custom bulk options, write down the options list in the text box (one option per line)
7. Then click on `Add New Choices` button. This should add the provided bulk options to the field currently active.
8. Now see if everything is working properly

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Add bulk choice
